stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/MessageDeletedViewHolder.kt

```diff
@@ -24,5 +26,9 @@ internal class MessageDeletedViewHolder(
         super.bindData(data, diff)
 
         style.textStyleMessageDeleted.apply(binding.deleteLabel)
+
+        binding.messageContainer.updateLayoutParams<ConstraintLayout.LayoutParams> {
+            horizontalBias = if (data.isTheirs) 0f else 1f
+        }
```

Comment:
I think we need to do this for the footnote view as well, seeing things like this when running the app

![image](https://user-images.githubusercontent.com/12054216/130228527-9abbf8ec-a3af-4dcc-8f01-aae56c466d2e.png)

Reply:
o.O